installer/cloudflow-environment/values.yaml

```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 tags:
+  targetEks: false
```

Comment:
@lightbend/cloudflow-team let's see whether we are really using these tags anymore. I don't think we have any cluster-specific conditionals in our Helm templates anymore and, if we do, we shouldn't need them.